### PR TITLE
docs: add sharer contract address

### PR DESCRIPTION
**Pull request by dudesahn**

### CODE REVIEW

File: docs/OPERATIONS.md [hunk 1]
```diff
@@ -124,6 +124,7 @@
 - Setup rewards for your strategy by calling `sharer.addContributors`.
 - Include devs if you forked someone else's strategy.
 - Be sure to reward people who helped you.
+- You can find the sharer here: https://etherscan.io/address/0x2c641e14afecb16b4aa6601a40ee60c3cc792f7d
```
Comment (fameal) [inline docs/OPERATIONS.md:127]: Can you add the link as [0x2c641e14afecb16b4aa6601a40ee60c3cc792f7d](https://etherscan.io/address/0x2c641e14afecb16b4aa6601a40ee60c3cc792f7d) ?

Comment (fubuloubu) [inline docs/OPERATIONS.md:127]:
```suggestion
- You can find the sharer here: [https://etherscan.io/address/0x2c641e14afecb16b4aa6601a40ee60c3cc792f7d](https://etherscan.io/address/0x2c641e14afecb16b4aa6601a40ee60c3cc792f7d)
```

Comment (fubuloubu) [reply docs/OPERATIONS.md:127]: per @fameal's comment https://github.com/yearn/yearn-vaults/pull/242#discussion_r596762338

**Review by fameal — APPROVED** [review]
LGTM

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**